src/core/public/chrome/ui/header/header.tsx

```diff
@@ -114,7 +114,7 @@ export function Header({
   const toggleCollapsibleNavRef = createRef<HTMLButtonElement & { euiAnimate: () => void }>();
   const navId = htmlIdGenerator()();
   const className = classnames('hide-for-sharing', 'headerGlobalNav');
-  const { useExpandedHeader = true, darkMode } = branding;
+  const { useExpandedHeader = true } = branding;
```

Comment:
Does it mean now breadcrumbs cannot be customized theme specific and is inherited as it is from oui?

Reply:
Yeah, breadcrumbs in OpenSearch Dashboards were never intended to be customized/customizable, we just had to do things that way because OUI was not yet available. But OUI should be responsible for the styling and theming of the breadcrumb components.